monitoring-plugins/check_openshift.py

```diff
@@ -393,7 +559,7 @@ def get_project_labels(self, required_labels):
 
     try:
         STATE = myos.os_STATE
-        OUTPUT_MESSAGE = myos.os_OUTPUT_MESSAGE
+        OUTPUT_MESSAGE = myos.os_OUTPUT_MESSAGE + '|' + myos.os_OUTPUT_MESSAGE_PERFDATA
```

Comment:
could you use this way to concat ?

`OUTPUT_MESSAGE = "%s | %s" % (myos.os_OUTPUT_MESSAGE, myos.os_OUTPUT_MESSAGE_PERFDATA)`